src/wwwroot/services/resources.js

```diff
@@ -71,12 +66,22 @@
     }
 
     function translate(resource) {
-      
       var lang = $translate.use();
       resourcesService.data[resource] = responses[resource].map(function(val){
         return { code: val.code, name: val[lang] };
       });
 
+      if(resource == 'countries'){
+        resourcesService.data[resource].forEach(element => {
+          var country = responses[resource].filter(c=>c.code==element.code);
+          if (typeof country[0].provinces !== 'undefined' && country[0].provinces.length > 0) {
+            element.provinces = country[0].provinces.map(function(val){
+              return { code: val.code, name: val[lang] };
```

Comment:
Si no me equivoco en este proyecto se están usando los espacios de la siguiente manera:

```suggestion
      if (resource == 'countries') {
        resourcesService.data[resource].forEach(element => {
          var country = responses[resource].filter(c => c.code == element.code);
          if (typeof country[0].provinces !== 'undefined' && country[0].provinces.length > 0) {
            element.provinces = country[0].provinces.map(function(val) {
              return { code: val.code, name: val[lang] };
```

Reply:
Ok, lo cambio

Reply:
hecho!